controllers/API/loginCode.js

```diff
@@ -19,14 +19,14 @@ router.post("/API/loginCode", auth.isAuthorised("ALTER_LOGIN_CODES"), function (
 router.get("/API/loginCode", function (req, res) {
 	const count = parseInt(req.query.count);
 	delete req.query.count;
-	if(req.query.code) {
+	if (req.query.code) {
```

Comment:
`"space-after-keywords": ["error", "never"]` this needs to be added to `eslintrc.js`